public/app/features/org/SelectOrgPage.tsx

```diff
@@ -18,29 +18,33 @@ const navModel = {
   },
 };
 
-const getUserOrgs = async () => {
-  return await getBackendSrv().get('/api/user/orgs');
-};
+function mapStateToProps(state: StoreState) {
+  return {
+    userOrgs: state.user.orgs,
+  };
+}
 
 const mapDispatchToProps = {
   setUserOrganization,
+  getUserOrganizations,
 };
 
-const connector = connect(null, mapDispatchToProps);
+const connector = connect(mapStateToProps, mapDispatchToProps);
 
 type Props = ConnectedProps<typeof connector>;
 
-export const SelectOrgPage: FC<Props> = ({ setUserOrganization }) => {
-  const [orgs, setOrgs] = useState<UserOrg[]>();
+export const SelectOrgPage: FC<Props> = ({ setUserOrganization, getUserOrganizations, userOrgs }) => {
+  const [orgs, setOrgs] = useState<UserOrg[]>(userOrgs);
 
   const setUserOrg = async (org: UserOrg) => {
     setUserOrganization(org.orgId);
     window.location.href = config.appSubUrl + '/';
   };
 
   useAsync(async () => {
-    setOrgs(await getUserOrgs());
+    setOrgs(await getUserOrganizations());
```

Comment:
For some reason, this doesn't work. Just investigated and noticed it was because I was using the `UserOrganizationsLoadedAction` which is a reducer in the admin directory. I suspect the reason is because the `admin` page is not in view and thus, it's actions/reducers won't work.

This is interesting! 😅 @ashharrison90